spring-data-geode/src/test/java/org/springframework/data/gemfire/function/execution/onservers/Metric.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2016-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.springframework.data.gemfire.function.execution.onservers;
```

Comment:
Move to `org.springframework.data.gemfire.function.sample`.

Reply:
Moved